apps/system/js/rocketbar.js

```diff
@@ -459,7 +459,10 @@
      * @memberof Rocketbar.prototype
      */
     _handle_home: function() {
-      if (this.isActive()) {
+      if (this._activateCall) {
```

Comment:
As we are doing this same check several times, can we extract it to a separate method?